data/json/move_modes.json

```diff
@@ -52,5 +52,22 @@
     "sound_multiplier": 0.5,
     "move_speed_multiplier": 0.5,
     "swim_speed_mod": 50
+  },
+  {
+    "type": "movement_mode",
+    "id": "lie_down",
+    "character": "l",
+    "panel_char": "L",
+    "name": "lie down",
+    "panel_color": "green",
+    "symbol_color": "blue",
+    "exertion_level": "NO_EXERCISE",
+    "change_good_none": "You lie down.",
+    "change_good_animal": "You stop your steed.",
+    "change_good_mech": "You turn off your mech's leg servos",
+    "move_type": "lying",
+    "sound_multiplier": 0,
```

Comment:
No sound at all feels too OP. I think something like 0.2 would be more appropriate.

Reply:
Changed